src/flow_solver/flow_solver.cpp

```diff
@@ -35,12 +35,20 @@ FlowSolver<dim, nstate>::FlowSolver(
 , final_time(flow_solver_param.final_time)
 , input_parameters_file_reference_copy_filename(flow_solver_param.restart_files_directory_name + std::string("/") + std::string("input_copy.prm"))
 , dg(DGFactory<dim,double>::create_discontinuous_galerkin(&all_param, poly_degree, poly_degree, grid_degree, flow_solver_case->generate_grid()))
-, ode_solver(ODE::ODESolverFactory<dim, double>::create_ODESolver(dg))
 {
     flow_solver_case->set_higher_order_grid(dg);
     dg->allocate_system();
+
+    if(flow_solver_param.reduced_order){
```

Comment:
Btw are you always gonna be passing an object of type `ProperOrthogonalDecomposition::OfflinePOD` to ODE solver ? because we could just have a POD factory that takes parameters and returns a `nullptr` if there's no POD being done, and then the POD object would be held by the `FlowSolver` class like `dg` and `ode_solver`

Reply:
I saw that you've changed the condition to use ODESolverEnum, looks good! Is this the only place where `flow_solver_param.reduced_order` is used? If so, it could be removed from the parameters list.

Reply:
Yeah that was the only use of it. I have removed it from the parameters list in the newest commit.

Reply:
Okay, sounds great!